src/codec_svt.c

```diff
@@ -73,7 +73,9 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
         }
         svt_config->encoder_color_format = color_format;
         svt_config->encoder_bit_depth = (uint8_t)image->depth;
+#if !SVT_AV1_CHECK_VERSION(0, 8, 8)
```

Comment:
Note that I suggest two changes in this comment.

1. Apply the following patch to this file. We need to provide a backup definition of the `SVT_AV1_CHECK_VERSION` macro so that this file can be compiled against a version of SVT-AV1 older than v0.9.0:

```
diff --git a/src/codec_svt.c b/src/codec_svt.c
index d7a7e88..32853fd 100644
--- a/src/codec_svt.c
+++ b/src/codec_svt.c
@@ -9,6 +9,18 @@
 
 #include <string.h>
 
+// The SVT_AV1_CHECK_VERSION macro was added in SVT-AV1 v0.9.0. Define this macro for older versions
+// of SVT-AV1.
+#ifndef SVT_AV1_CHECK_VERSION
+// clang-format off
+#define SVT_AV1_CHECK_VERSION(major, minor, patch)                            \
+    (SVT_AV1_VERSION_MAJOR > (major) ||                                       \
+     (SVT_AV1_VERSION_MAJOR == (major) && SVT_AV1_VERSION_MINOR > (minor)) || \
+     (SVT_AV1_VERSION_MAJOR == (major) && SVT_AV1_VERSION_MINOR == (minor) && \
+      SVT_AV1_VERSION_PATCHLEVEL >= (patch)))
+// clang-format on
+#endif
+
 #define STR_HELPER(x) #x
 #define STR(x) STR_HELPER(x)
 #define SVT_FULL_VERSION STR(SVT_VERSION_MAJOR) "." STR(SVT_VERSION_MINOR) "." STR(SVT_VERSION_PATCHLEVEL)

```

2. Change this line to

`#if !SVT_AV1_CHECK_VERSION(0, 9, 0)`

Reply:
Jamaika1 : You didn't add the code that defines the SVT_AV1_CHECK_VERSION macro for older versions of SVT-AV1. We need that code to compile against an installed SVT-AV1, which may be older than v0.9.0. I took care of that in https://github.com/AOMediaCodec/libavif/pull/823.